tests/test_consumer.py

```diff
@@ -138,6 +137,18 @@ async def test_consume_with_resubscribe(stream: str, consumer: Consumer, produce
     await wait_for(lambda: len(captured) >= 2)
     assert captured == [b"one", b"two"]
 
+async def test_consume_with_resubscribe_on_last(stream: str, consumer: Consumer, producer: Producer) -> None:
```

Comment:
Seems this test will pass for both LAST and FIRST. I'm curious if we could write a test that would reliably pass only for LAST

Reply:
Yeah you're right this test doesn't belong here any longer. The one assert that seems to actually test LAST might be here https://github.com/qweeze/rstream/pull/25/files#diff-b6b23775d81e14e919519e377f7d4c6f9bcd28db05bfbd5645492a9a2f18cd81R90

Reply:
Is there something with query_offset we can try